empathyco/x-archetype
# feat(currency): add currencyFormat injection

# Description

- Update X-Components version.
- Add implementation of the new 'currencyFormat' injection, injecting from the App the current currency (from the reactive snippetConfig) format.
- Add initial currency configuration for EUR and USD: this is supposed to be done in an intermediate project which is not created yet.
- Modify `Result` in order to allow modifying a new slot called `price` so the format can easily be overridden.

# Test

- Check that all the requirements are fulfilled and everything work as expected.
- Check that you can modify the reactive configuration from the console: `window.initX.currency = "USD"` and see how the format changes.
